### ISSUE TITLE
Add timeframe to exporter importer

### ISSUE THREAD

**Issue by edrock200** [Casvt/Plex-scripts#191]
In what file should the feature be added?
plex_exporter_importer.py

What is the feature request?
Use the "data added" value to be able to export only after or before specific date. Presently exporting and importing intros takes a significant amount of time because it's doing all intros. But it would be nice to have a scheduled cron that exports say, the last week, and imports the last week to minimize the time for the transaction. 

Tool is awesome btw, thanks for making it.

**Comment by Casvt** [comment]
Already implemented.

When running the script for export, supply the `-L` flag and as it's value the file path to the database file from the previous export. The script will update that database file, only exporting stuff that has been changed since the last time.
